examples/send-and-receive-emails/send-and-receive-emails.description

```diff
@@ -0,0 +1,4 @@
+// The Email Connector is used to send (with SMTP) and receive (with POP3 or
```

Comment:
The names of the description files have to be the same as the directory names, with `-`s replaced by `_`s.

Reply:
Added PR, https://github.com/ballerina-platform/ballerina-lang/pull/21822